source/Ship.cpp

```diff
@@ -180,14 +180,14 @@ namespace {
 	// delaying their firing for another reload cycle. The less energy
```

Comment:
First line of this comment still refers to ionization. (Can't reach it from here.)